app/controllers/projects_controller.rb

```diff
@@ -343,6 +343,6 @@ def return_context
     render_404 && return if @project.blank?
     return unless @project.deleted?
     flash[:alert] = 'The project you requested had been deleted.'
-    redirect_to user_path(@user)
+    check_redirect "/#{@user.username}"
```

Comment:
Hey, you can sign up for hakiri, follow your branch on your own repo and you will be able to see if you have any warnings on that branch. It is free for pubic repos. Don't forget to confirm if all the redirect issues are solved, we can close this one.